sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -158,6 +164,12 @@ def test_regressor_parameter_checks():
                          " or an integer. 'invalid' was passed",
                          GradientBoostingRegressor(n_iter_no_change='invalid')
                          .fit, X, y)
+    assert_raise_message(ValueError, "Presort must be a boolean",
+                         GradientBoostingRegressor(presort=5.324)
```

Comment:
It might be better not to use such a magical value, maybe presort='invalid' like before is fine.